src/lib/components/ResourceTypeField.js

```diff
@@ -103,11 +103,10 @@ export class ResourceTypeField extends Component {
         error={this.groupErrors(form.errors, fieldPath)}
         id={fieldPath}
         label={
-          <FieldLabel
-            htmlFor={fieldPath}
-            icon={labelIcon}
-            label={label}
-          />
+          <label htmlFor={fieldPath}>
+          {labelIcon ? <Icon name={labelIcon} /> : null}
+          {label}
+        </label>
```

Comment:
The problem is that this "exception" will come up again for Languages and Identifiers and who knows what other fields we may get in the future. Perhaps a more encompassing strategy:

1- See if there is a css rule pattern that can target labels of subfields (`.fielfd > .field label` or something like it) to override those.
2- If there isn't a nice pattern like this across the fields, then make the react-invenio-forms `FieldLabel` take a prop to specify if it's a top-level field or not. The default being the most common current case, so that we don't have to change every instance (Like you were saying, I think that default is it being used for top-level fields.)
?

If that's too vague, let's have a chat about it :smile: